bin/check-sqs-messages.rb

```diff
@@ -121,9 +128,8 @@ def run
       crit = false
       queues = []
 
-      sqs.queues.with_prefix(config[:prefix]).each do |q|
-        queue_name = q.arn.split(':').last
-        messages = q.approximate_number_of_messages
+      sqs.queues(queue_name_prefix: config[:prefix]).each do |q|
+        messages = sqs.client.get_queue_attributes(queue_url: q.url, attribute_names: ['All']).attributes[config[:metric]].to_i
 
```

Comment:
@eheydrick can you give me your example command you are running as I am not seeing that locally when I test:

Testing full queue name:
```
$ bundle exec ./bin/check-sqs-messages.rb -r us-west-2 -q dev-aws-cleaner -w 1000 -c 1200 -m ApproximateAgeOfOldestMessage
SQSMsgs OK: 0 messages in dev-aws-cleaner queue
```

Testing prefix
```
$ bundle exec ./bin/check-sqs-messages.rb -r us-west-2 -p dev -w 1000 -c 1200 -m ApproximateAgeOfOldestMessage
SQSMsgs OK: All queues matching prefix 'dev' OK
```